client/me/purchases/manage-purchase/index.tsx

```diff
@@ -1181,7 +1184,7 @@ class ManagePurchase extends Component<
 								<div className="manage-purchase__contact-partner">
 									{ translate( 'Please contact your site host %(partnerName)s for details', {
 										args: {
-											partnerName: getPartnerName( purchase ),
+											partnerName: getPartnerName( purchase ) ?? '',
```

Comment:
I think that's a reasonable fallback. Perhaps it would be nice to fix the case of double spacing when partner name is nullish.

Reply:
Do you have a recommendation on how to do that without creating a new string?